add paragraph about p.a.layout being installable (#2078)
plone/documentation: docs/backend/upgrading/version-specific-migration/upgrade-to-62.md

* `plone.locking`
* `plone.protect`

> Starting with Plone 6.2, you can choose to activate plone.app.layout in the Add-ons control panel.

Reviewer: ```suggestion
Starting with Plone 6.2, you can activate plone.app.layout in the {guilabel}`Add-ons` control panel.
```

> If you use the Classic UI, you should do this. If you use Volto as your (main) UI, you should not do this.

Reviewer: ```suggestion
If you use Classic UI, you should activate it.
If you use Volto, you should not activate it.
```

> The automatic upgrade tries to do the right thing, but you should check the Add-ons control panel.

Reviewer: I'm not sure what is the "right thing", so I made a best guess. Please verify.

```suggestion
The automatic upgrade attempts to activate plone.app.layout according to your original configuration, but you should check the {guilabel}`Add-ons` control panel to make sure.
```

> The setting currently has little influence, so if you are not sure: don't worry about it.

Reviewer: This statement causes anxiety which it attempts to prevent. What influence does activation have? If I'm "not sure" about what? Maybe just remove this sentence?

```suggestion
The setting currently has little influence, so if you're not sure, then don't worry about it.
```

Reviewer: I still have those questions. 😁

Author: Yes, you are right, we should just remove it.

Reviewer 2: I was about to suggest this:

> The setting currently has little influence: one warning page from `plone.protect` shows a simpler layout.

But leaving it out is fine.
Maybe remove the many lines with only white space though.

* `plone.locking`
* `plone.protect`

Starting with Plone 6.2, you can activate plone.app.layout in the {guilabel}`Add-ons` control panel.

If you use Classic UI, you should activate it.
If you use Volto, you should not activate it.

The automatic upgrade attempts to activate plone.app.layout according to your original configuration, but you should check the {guilabel}`Add-ons` control panel to make sure.

> 

Reviewer: ```suggestion
```

> 

Author: ```suggestion

```
